pennylane/devices/qutrit_mixed/measure.py

```diff
@@ -238,16 +275,25 @@ def get_measurement_function(
 
 
 def measure(
-    measurementprocess: MeasurementProcess, state: TensorLike, is_state_batched: bool = False
+    measurementprocess: MeasurementProcess,
+    state: TensorLike,
+    is_state_batched: bool = False,
+    measurement_errors=None,
 ) -> TensorLike:
     """Apply a measurement process to a state.
 
     Args:
         measurementprocess (MeasurementProcess): measurement process to apply to the state.
         state (TensorLike): the state to measure.
         is_state_batched (bool): whether the state is batched or not.
+        measurement_errors (List[Callable]): List of operators to apply to each wire being measured
+        to simulate measurement error.
 
     Returns:
         Tensorlike: the result of the measurement process being applied to the state.
     """
-    return get_measurement_function(measurementprocess)(measurementprocess, state, is_state_batched)
+    measurement_function = get_measurement_function(measurementprocess)
+
+    if measurement_function is calculate_reduced_density_matrix:
+        return measurement_function(measurementprocess, state, is_state_batched)
+    return measurement_function(measurementprocess, state, is_state_batched, measurement_errors)
```

Comment:
suggestion: does it make sense to just add an optional argument to `get_measurement_function` in order to remove the need for the conditional?

Reply:
I thought about that, but it required a ton of `fuctools.partial`, I think it may be better to just remove `get_measurement_function`, what do you think?